src/routes/Manual.svelte

```diff
@@ -104,14 +105,43 @@ $: lfDutyCycle = 50;
 $: hfPeriod = 250;
 $: hfDutyCycle = 50;
 
-$: intensity = 100;
+$: intensity = 100; 
+
+$: single_pulse_block = [0,0,0,0];
+$: hfOn = hfPeriod * (hfDutyCycle/100);
+$: hf_block = [hfOn & 0x000000ff, (hfOn & 0x0000ff00) >> 8, hfPeriod & 0x000000ff,(hfPeriod & 0x0000ff00) >> 8];
+$: lfOn = lfPeriod * (lfDutyCycle/100);
+$: lf_block = [lfOn & 0x000000ff, (lfOn & 0x0000ff00) >> 8, lfPeriod & 0x000000ff, (lfPeriod & 0x0000ff00) >> 8];
 
 // MARK: Device Input
 
 $: activeHexagon = -1;
 $: mouseDown = false;
 $: pendingTimeout = false;
 
+let block0_31 = [0,0,0,0];
+let block32_63 = [0,0,0,0];
+let block64_95 = [0,0,0,0];
+let block96_127 = [0,0,0,0];
+
+
+//Leaving this reactive 
+$: act_command = `{ "ActuatorsCommand": {
+    "fabric_name": "${devices[activeDevice]}",
```

Comment:
There was a small bug in the React version where device/fabric name would have white space removed, effectively changing the name we submit. It would reduce network traffic very marginally but caused a mismatch on whitespace. 

When configuring device names, I would recommend trimming leading and trailing spaces and replace the remaining white space with underscores before creating the command to send to the CLI.